src/components/Card.jsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/prop-types */
+import { Link } from "react-router-dom";
+import { Poster } from "../components/Poster";
+import "../pages/MovieList.css";
+import { useState } from "react";
+
+export const Card = ({ movie }) => {
+  const [isHovered, setIsHovered] = useState(false);
+  const handleMouseOver = () => {
+    setIsHovered(true);
+  };
+  const handldeMouseLeave = () => {
+    setIsHovered(false);
+  };
```

Comment:
I liked that you managed the hovering with functions instead of simple CSS! That's a great tip!